charts/azure-managed-cluster/README.md

```diff
@@ -0,0 +1,154 @@
+# CAPI Azure
+
+## Prerequisites
+- [Docker Desktop](https://www.docker.com/)
+- [Kind](https://kind.sigs.k8s.io/)
+- [ClusterCTL](https://cluster-api.sigs.k8s.io/clusterctl/overview.html) Version v1.6.1 or older
+- [Helm](https://helm.sh) version v3.14.0 or later
+
+## Prerequisites Installations
+- Docker Desktop
+ https://www.docker.com/products/docker-desktop 
+
+- Install Kind
+  https://kind.sigs.k8s.io/
+
+- Install Clusterctl
+  https://cluster-api.sigs.k8s.io/clusterctl/overview.html
+
+- Install Helm3
+- https://helm.sh/docs/intro/install/
+
+## Deploy
+
+Clone this repo
+
+```bash
+git clone https://github.com/ams0/azure-managed-cluster-capz-helm.git
+cd azure-managed-cluster-capz-helm
+
+```
+
+Create an Azure Service Principal or Managed Identity
+
+- Follow the steps on this link to create a service principal <https://capz.sigs.k8s.io/topics/identities#service-principal>
+- Follow the steps on this link to create a managed identity <https://capz.sigs.k8s.io/topics/identities#managed-identity>
+
+Create and Update Environment variables
+Edit your ~/.bashrc file to include Azure Service Principal environment variables or just export on your current terminal session
+```bash
+export AZURE_CLIENT_ID="" (if using Service Principal)
+
+export AZURE_CLIENT_SECRET="" (if using Service Principal)
+
+export AZURE_SUBSCRIPTION_ID=""
+
+export AZURE_TENANT_ID=""
+
+```
+
+Load Environment variables
+```bash
+source clusterctl.env
```

Comment:
This is not provided in the PR; it probably refer to https://github.com/ams0/azure-managed-cluster-capz-helm/blob/main/clusterctl.env but not sure the values are still valid. I'm testing this as I write.